### PR TITLE
docs(ui-modal,ui-tray): function based examples are added to Modal an…

### PR DESCRIPTION
**Pull request by ToMESSKa**
…d Tray

Closes: INSTUI-4149

### CODE REVIEW

File: packages/ui-tray/src/Tray/README.md [hunk 1]
```diff
@@ -6,27 +6,87 @@ The Tray is an actionable container that is triggered by click and does not need
...
+    }
+
+    render() {
+      return (
+        <div style={{ padding: '0 0 16rem 0', margin: '0 auto' }}>
+          <Button
+            onClick={() => {
+              this.setState({ open: true })
+            }}
+            ref={(c) => (this._showButton = c)}
```
Comment (matyasf) [inline packages/ui-tray/src/Tray/README.md:50]: this doesnt seem to do anything, you can delete it

File: packages/ui-tray/src/Tray/README.md [hunk 2]
```diff
@@ -6,27 +6,87 @@ The Tray is an actionable container that is triggered by click and does not need
...
-    return (
+  render(<Example />)
+  ```
+
+- ```js
+  const FPO = lorem.paragraph()
+
+  const Example = () => {
+    const [open, setOpen] = useState(false)
+    const showButtonRef = useRef(null)
```
Comment (matyasf) [inline packages/ui-tray/src/Tray/README.md:83]: you can delete this one too

**Review by matyasf — CHANGES_REQUESTED** [review]

**Review by HerrTopi — APPROVED** [review]

File: packages/ui-tray/src/Tray/README.md (hunk 1 above)
Comment (ToMESSKa) [reply packages/ui-tray/src/Tray/README.md]: Done

File: packages/ui-tray/src/Tray/README.md (hunk 2 above)
Comment (ToMESSKa) [reply packages/ui-tray/src/Tray/README.md]: Done

**Review by matyasf — APPROVED** [review]

**Pull request merged**